doc/guide/cockpit-http.xml

```diff
@@ -27,7 +27,15 @@ http = cockpit.http(options)
         <listitem><para>Connect to an address other than localhost. Must be a valid host name or IP address.
           To use this option you also must provide a port number.</para></listitem>
       </varlistentry>
-
+      <varlistentry>
+        <term><code>"port"</code></term>
+        <listitem><para>Port number to use with "address" option.</para></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><code>"tls"</code></term>
+        <listitem><para>Object properties for an https connection. See
+        <code><link url="https://github.com/cockpit-project/cockpit/blob/master/doc/protocol.md#payload-http-stream2">http-stream2 TLS options</link></code>.</para></listitem>
```

Comment:
The doc parser doesn't like this. I think its needs to use ```<ulink url...>```

```
/home/runner/cockpit/./doc/guide/cockpit-guide.xml:37: element link: validity error : Element link does not carry attribute linkend
/home/runner/cockpit/./doc/guide/cockpit-guide.xml:37: element link: validity error : No declaration for attribute url of element link
```

Reply:
Fixed.